CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing to COVID-19 Resources India
+
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
```

Comment:
`We use GitHub to host code, track issues, and feature requests, as well as accept pull requests.`